crypto/ledger/ledger_test.go

```diff
@@ -102,26 +102,15 @@ func TestPublicKeySafe(t *testing.T) {
 
 	require.NoError(t, err)
 	require.NotNil(t, priv)
-
 	require.Nil(t, ShowAddress(path, priv.PubKey(), sdk.GetConfig().GetBech32AccountAddrPrefix()))
-
-	require.Equal(t, "eb5ae98721034fef9cd7c4c63588d3b03feb5281b9d232cba34d6f3d71aee59211ffbfe1fe87",
-		fmt.Sprintf("%x", cdc.Amino.MustMarshalBinaryBare(priv.PubKey())),
-		"Is your device using test mnemonic: %s ?", testutil.TestMnemonic)
-
-	pubKeyAddr, err := sdk.Bech32ifyPubKey(sdk.Bech32PubKeyTypeAccPub, priv.PubKey())
-	require.NoError(t, err)
-	require.Equal(t, "cosmospub1addwnpepqd87l8xhcnrrtzxnkql7k55ph8fr9jarf4hn6udwukfprlalu8lgw0urza0",
-		pubKeyAddr, "Is your device using test mnemonic: %s ?", testutil.TestMnemonic)
-
-	require.Equal(t, "cosmos1w34k53py5v5xyluazqpq65agyajavep2rflq6h",
-		addr, "Is your device using test mnemonic: %s ?", testutil.TestMnemonic)
+	checkDefaultPubKey(t, priv)
 
 	addr2 := sdk.AccAddress(priv.PubKey().Address()).String()
 	require.Equal(t, addr, addr2)
 }
 
 func TestPublicKeyHDPath(t *testing.T) {
+	// TODO: convert this tests to not use bech32
```

Comment:
should we add an issue for that or reference an existing issue here?

Reply:
In fact I'm solving it here.

Reply:
I'm removing the todo, but we should run the test with ledger support. What's the best way to do it?

Reply:
what do you mean exactly by "with ledger support"?

Reply:
if we can have a test with a real ledger

Reply:
I will update this test in a separate PR.

Reply:
ok do we have an issue to track that?